src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -4727,13 +4727,11 @@ Copyright (C) Microsoft Corporation. All rights reserved.
                                         GetCopyToOutputDirectoryItems
 
     Get all project items that may need to be transferred to the output directory.
-    This includes baggage items from transitively referenced projects. It would appear
-    that this target computes full transitive closure of content items for all referenced
-    projects; however that is not the case. It only collects the content items from its
-    immediate children and not children of children. The reason this happens is that
-    the ProjectReferenceWithConfiguration list that is consumed by _SplitProjectReferencesByFileExistence
-    is only populated in the current project and is empty in the children. The empty list
-    causes _MSBuildProjectReferenceExistent to be empty and terminates the recursion.
+    This includes baggage items from transitively referenced projects.
+
+    As of 17.0, content items are copied transitively by default.
+    Set `MSBuildCopyContentTransitively` to false to opt out.
+    See https://github.com/dotnet/msbuild/pull/6622 for more info.
```

Comment:
tiny nit:
```suggestion
    See https://github.com/dotnet/msbuild/pull/6622 for more details.
```

Reply:
We should probably have a doc page for this. Can you work with @ghogen to create one/make sure it's up to date?

Reply:
the documentation is still outdated [here](https://learn.microsoft.com/en-us/visualstudio/msbuild/msbuild-targets?view=vs-2022). @eerhardt @ghogen Could you update the documentation?